### ISSUE TITLE
Can the implementaiton be upgraded to opentelemetry ?

### ISSUE THREAD

**Issue by ovidiubuligan** [Medialogia/OpenTracing.Contrib.Wcf#1]
How big of an effort is required to upgrade to open telemetry ?

**Comment by iEmiya** [comment]
I don't think it will be difficult to upgrade OpenTelemetry, but within framework of this project, changes are not planned in near future.

Сurrent solution is used in industrial operation and the simple use of new framework will not give such an extensive feedback, besides, WCF itself is being forgotten.

Look towards OpenTracing.Contrib.Grpc
